### ISSUE TITLE
Need support for output parameters when executing stored procedures

### ISSUE THREAD

**Issue by invalidred** [anvk/easy-sql-tests#3]
Currently _executeStoredProc()'s callback only returns error and recordsets. We also need to return a third parameter, which is an object that contains all the output parameters.

To accomplish this we will need the following:

1) _executeStoredProc needs args {} that will need to specify if they are output params, by default they will be input parameter

2) when request.execute is called, we will need to pass the callback in addition to error and recordsets an object that contains output parameters.

**Comment by anvk** [comment]
Yeah that would be a good addition. Maybe let's add it once other pieces are in place.
